index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="Style.css">
+</head>
+<body>
+	<nav>
+		<label class="logo">My Logo</label>
+		<ul>
+			<li><a href="#" class="bar">
+			<div class="b1"></div>
+			<div class="b2"></div>
+			<div class="b3"></div>
+		</a></li>
+			</ul>	
+		</nav>
+		<div class="backimg">
+		<header>
+			<h2>I’m Mariane<br>Glad to see you!</h2>
+			<p class="p1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+			<p class="p2">LET’S CONNECT</p>
+			<div class="images">
+				<img src="img/Vector.png" alt="icon1">
+				<img src="img/Linkedin.png" alt="icon2">
+				<img src="img/Vector(1).png" alt="icon3">
+				<img src="img/Vector(2).png" alt="icon4">
+				<img src="img/Vector(3).png" alt="icon5">
+			</div>
```

Comment:
- [ ] For the list of icons you should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). You can check the following example to get an idea. Please work on that. :writing_hand: 

```
<ul>
  <li><a href="https://github.com/mgmediaweb" target="new"><i class="fab fa-github social-icon"></i></a></li>
</ul>
```

Reply:
> * [ ]  For the list of icons you should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). You can check the following example to get an idea. Please work on that. writing_hand
> 
> ```
> <ul>
>   <li><a href="https://github.com/mgmediaweb" target="new"><i class="fab fa-github social-icon"></i></a></li>
> </ul>
> ```

Please work on the suggested changes. :writing_hand: